app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class ReportsController < BaseTraineeController
+  def index
+    @current_user = current_user
+  end
+
+  def itt_new_starter_data_sign_off
+    respond_to do |format|
+      format.html do
+        @census_date = census_date(AcademicCycle.current.start_year).strftime("%d %B %Y")
+      end
+      format.csv do
+        authorize(:trainee, :export?)
+        send_data(data_for_export.data, filename: filename, disposition: :attachment)
+      end
+    end
+  end
+
+private
+
+  def data_for_export
+    @data_for_export ||= Exports::TraineeSearchData.new(policy_scope(IttNewStarterDataSignoffTraineesService.new(census_date(AcademicCycle.current.start_year)).call))
+  end
+
+  def filename
+    current_time = Time.zone.now
+    "#{current_time.strftime('%F_%H_%M_%S')}_New-trainees-#{AcademicCycle.current.start_year}-#{AcademicCycle.current.end_year}-sign-off-Register-trainee-teachers_exported_records.csv"
+  end
+
+  def census_date(year)
+    date_of_nth_weekday(10, year, 3, 2)
+  end
+
+  def date_of_nth_weekday(month, year, wday, nth)
+    base_date = Date.new(year, month)
```

Comment:
Yeah so the 2nd wednesday of every month is hard to calculate. I really wasnt sure where this code should live as it could (maybe!?) be useful again. If you think itd be useful we could go over it together? I think ive named the variables badly but i couldnt think of anything better =P